Gems/Atom/RPI/Code/Source/RPI.Public/Pass/AttachmentReadback.cpp

```diff
@@ -289,9 +289,8 @@ namespace AZ
 
             m_dispatchItem.m_arguments = dispatchArgs;
 
-            const RHI::SingleDeviceImageView* imageView = context.GetImageView(m_attachmentId);
+            const RHI::MultiDeviceImageView* imageView = context.GetImageView(m_attachmentId);
```

Comment:
I personally prefer already converting these to `const auto*` while you are changing these lines.